source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -33,12 +33,24 @@ You can create a Prometheus service within your PaaS space and allow it to bind
   - `cf create-service gds-prometheus prometheus <service-instance-name>`
 2. Either [update your app's manifest.yml] to bind your new service or bind it using the CLI
   - `cf bind-service <app-name> <service-instance-name>`
+3. If your app uses a custom domain, you need to make sure that the [authorization header and other app headers][] are being forwarded otherwise the basic auth to the metrics endpoint will fail and your app won't have access to your other headers.
+  - `cf update-service <cdn-route-service> \
+    -c '{"domain": "<custom-domain>", "headers": ["Accept", "Authorization", "<other app header values>"]}'`
 
-Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana](https://grafana-paas.cloudapps.digital).
+Within 5 minutes Prometheus will start scraping your application for metrics, you can validate this by checking [Grafana][].
 
+### Notes about the app route
+
+* Only the first route of the app will be scraped.
```

Comment:
OK, so drop the heading and just have it as a lead-in line, the bullets apply to all domains, regardless if they are custom or not.

Reply:
There is a possibility that apps will have multiple routes especially if they are using custom domains, so they might have something like this - 

`custom.domain.gov.uk, an-interesting-app.cloudapps.digital`, so only the first route will be picked up. 

I've got no idea how common it would be for apps to have this, but the behaviour of the service broker is such that it will ignore any routes after the first one so it's probably a good idea to let a user know in case they do have multiple routes.